plugins/funind/glob_term_to_relation.ml

```diff
@@ -1161,7 +1174,8 @@ let rec rebuild_cons env nb_args relname args crossed_types depth rt =
 	  let evd = (Evd.from_env env) in
 	  let t',ctx = Pretyping.understand env evd t in
 	  let evd = Evd.from_ctx ctx in
-	  let type_t' = Typing.unsafe_type_of env evd (EConstr.of_constr t') in
+          let type_t' = Typing.unsafe_type_of env evd t' in
```

Comment:
We should have `Typing.NoEvars.etc` or some such system for when ignoring the possibility of evars makes sense. Maybe that's beyond the scope of this PR though.